Assets/Plugins/Pixel Crushers/Dialogue System/Scripts/Editor/Options/Timeline/DummyTimeLine.playable

```diff
@@ -0,0 +1,295 @@
+%YAML 1.1
```

Comment:
Can we put the playables in "Assets/ImportedAssets/Timeline" (would have to create a folder for them)

Or somewhere else with the other animations. The Plugins folder is really for the plugin code, rather than our own assets.